pkg/storage/export/export/export_test.go

```diff
@@ -74,6 +74,7 @@ const (
 	testNamespace  = "default"
 	ingressSecret  = "ingress-secret"
 	currentVersion = "v1beta1"
+	vmExportName   = "test"
```

Comment:
I would omit this new var to keep it consistent with the 1.4 backport https://github.com/kubevirt/kubevirt/pull/15176

Reply:
ok will remove it. Thanks!

Reply:
done. also updated 1.2 backport pr https://github.com/kubevirt/kubevirt/pull/15362